Week-6/Exercise6_5.js

```diff
@@ -0,0 +1,22 @@
+function findPair(arr, n) {
```

Comment:
rejecting this solution because:
This will work only on sorted arrays and works for array which contains pairs where smaller values is coming before larger value in the array. for example:  it works for `var arr = [10, 30, 20, 40];` but don't work for `var arr = [40, 30, 20, 10];` where I run `findPair(arr, 30)`